messages/jsonschema/Pickle.json

```diff
@@ -25,6 +25,8 @@
       "required": [
         "astNodeIds",
         "id",
+        "keyword",
```

Comment:
What do we need the `keyword` for? Isn't the `keywordType` sufficient?

Reply:
@sebrose said:

> By converting the actual keyword used in the scenario into G/W/T in the pickle, we lose the ability to reconstitute the scenario as written.

We have already lost the ability to reconstitute the scenario as written from a pickle, so this is a moot point (pickles inline backgrounds, examples tables etc). We never do this either. If we want to reconstitute the source, we'll do it from the `GherkinDocument` AST, not the pickle.

If we *really* want to obtain the original source keyword, we can do this by looking up the AST step from the [pickle step](https://github.com/cucumber/common/blob/main/messages/messages.md#picklestep)'s `astNodeIds`.

I think we should leave it out for now, and only add it later if we have a specific need for it.

Reply:
`keyword` is no longer in this change.